app/views/items/_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for [user, item] do |f| %>
+  <div class="form-group">
+    <%= f.label :description %>
+    <%= f.text_field :description, class: 'form-control', placeholder: "Enter the task" %>
```

Comment:
dangit ... yeah, I have "name" in the schema file.  I recall changing the
schema file is a no-no but can I edit the migration file?  Or is it safer
to leave that alone and change my `_form.html.erb` file?

Thanks

On Thu, Jan 7, 2016 at 1:14 PM, bjblock <notifications@github.com> wrote:

> In app/views/items/_form.html.erb
> <https://github.com/aaronkoz74/blocitoff/pull/5#discussion_r49105049>:
>
> > @@ -0,0 +1,7 @@
> > +<%= form_for [user, item] do |f| %>
> > +  <div class="form-group">
> > +    <%= f.label :description %>
> > +    <%= f.text_field :description, class: 'form-control', placeholder: "Enter the task" %>
>
> I'm assuming this is where the bug is.
>
> Take a look at your schema.rb file. Notice that items don't have a column
> called "description". So the problem is caused because you have a form
> input field for an attribute the the model doesn't know about.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aaronkoz74/blocitoff/pull/5/files#r49105049>.
>